assets/js/like.js

```diff
@@ -0,0 +1,65 @@
+jQuery(document).ready( function() {
+
+   jQuery(".bfc-post-like").click( function(event) {
+		var this2 = this
```

Comment:
I don't know if you want to be bothered with such things, but there are various places in this file where the indents aren't right or semicolons are missing. Neither cause functional problems, but it's still good hygiene to keep everything tidy.

Reply:
Thanks for catching the semicolons. I was only able to find one indent issue, with` jQuery(".bfc-post-like").click( function(event) {`

I agree. Good to keep it tidy.

Reply:
Cool. Here's what I'm seeing on the indent front (I'm assuming you are going for two space indents):

Line 3 has three spaces where it should have just two.

Lines 5 through 7 have six spaces when they should have four.

Looks like there are some tabs mixed in with the spaces… it's generally best practice to stick with one or the other. I personally prefer spaces. Depending on what editor you're using, you should be able to get it to do it automatically for you. 

The rest of it looks good :-)

Reply:
Thanks for this next level of catching these. Ah, the perils of copying code from elsewhere. We do have an editorconfig file which is set to indent_style = tabs and indent_size = 4. I've cleaned up my local file to that standard and will upload it in due course.

Reply:
Sounds good!